src/components/MobileSideMenu/WrappedMobileSideMenu.js

```diff
@@ -0,0 +1,35 @@
+import React, { Fragment, useState } from 'react';
+import MobileSideMenu from 'components/MobileSideMenu/index';
+
+const WrappedMobileSideMenu = () => {
```

Comment:
is this can actual component that should be in the library, or is this for testing only?

Reply:
This looks like it's only being used for the story- we should probably not have this exported with the rest of the code

Reply:
done